tests/scenarios/package.json

```diff
@@ -81,6 +82,7 @@
     "ember-source-latest": "npm:ember-source@latest",
     "ember-truth-helpers": "^3.0.0",
     "execa": "^5.1.1",
+    "node-fetch": "2.7.0",
```

Comment:
Using the latest version `3.3.2` triggers an error `Unknown file extension ".ts"`. When downgrading `node-fetch` to < 3, I don't have the issue and `fetch` runs fine.